### PR TITLE
fix: add Priyanshu Indra

### PR DESCRIPTION
**Pull request by PriyanshuIndra**
Hi! I am new to this community and this is my first PR. I have added my profile as discussed on the READ.me file.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/686"></a>

### CODE REVIEW

File: public/data/PriyanshuIndra.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "bio": "Student",
+    "avatar": "https://avatars.githubusercontent.com/u/74089982?v=4",
+    "links": [
+        {
+            "name": "Follow me on Twitter",
+            "url": "https://twitter.com/PriyanshuIndra4",
+            "icon": "twitter"
+        }
+    ]
+}
```
Comment (krishguptadev) [inline public/data/PriyanshuIndra.json:12]:
```suggestion
}

```

File: public/data/PriyanshuIndra.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Priyanshu Indra",
+    "bio": "Student",
+    "avatar": "https://avatars.githubusercontent.com/u/74089982?v=4",
```
Comment (krishguptadev) [inline public/data/PriyanshuIndra.json:4]:
```suggestion
    "avatar": "https://github.com/PriyanshuIndra.png",
```

File: generate.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// jshint esversion:6
```
Comment (krishguptadev) [inline generate.js:1]:
```suggestion
```

File: generate.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
+
```
Comment (krishguptadev) [inline generate.js:2]:
```suggestion
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]
Please revert package-lock.json.

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Please remove the package-lock file and generate.us file from your pull request.

File: public/data/PriyanshuIndra.json (hunk 1 above)
Comment (PriyanshuIndra) [reply public/data/PriyanshuIndra.json:12]: I don't understand by revert !! do you mean deleting brackets ??

File: generate.js (hunk 3 above)
Comment (PriyanshuIndra) [reply generate.js]: I resolved that, thank you !! I added because it was giving me error symbols.

File: public/data/PriyanshuIndra.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "Priyanshu Indra",
+    "bio": "Student",
+    "avatar": "https://github.com/PriyanshuIndra.png",
+    "links": [
+        {
```
Comment (Panquesito7) [inline public/data/PriyanshuIndra.json:6]: You can also add your GitHub profile. 😄
```suggestion
        {
            "name": "Follow me on GitHub",
            "url": "https://github.com/PriyanshuIndra",
            "icon": "github"
        },
        {
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please revert the changes in the `package-lock.json` file.

File: public/data/PriyanshuIndra.json (hunk 5 above)
Comment (PriyanshuIndra) [reply public/data/PriyanshuIndra.json:6]: Yep, thanks !! Going to do that 😊

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Looks good 👍 except for the `package-lock.json` file changes. Please revert.

**Comment by PriyanshuIndra** [comment]
> Looks good 👍 except for the `package-lock.json` file changes. Please revert.

I don't understand how to solve that, should I update the package ? Can you please help me ??

**Comment by Panquesito7** [comment]
> > Looks good 👍 except for the `package-lock.json` file changes. Please revert.
> 
> I don't understand how to solve that, should I update the package ? Can you please help me ??

Just copy the contents from the original file from the original repository, commit, and submit your changes. 🙂

File: public/data/PriyanshuIndra.json (hunk 2 above)
Comment (PriyanshuIndra) [reply public/data/PriyanshuIndra.json]: I resolved that. Thanks for helping.

File: generate.js (hunk 3 above)
Comment (PriyanshuIndra) [reply generate.js]: I don't know how to revert json-lock file.

**Review by Panquesito7 — APPROVED** [review]

**Review by krishguptadev — COMMENTED** [review]
LGTM!

**Pull request merged**